### PR TITLE
move LinkText & update imports

### PR DESCRIPTION
**Pull request by patrinoua**
The type of this PR is: **feat**

move LinkText & update imports

This PR resolves []

### PR Checklist (tick all before merging)

- [ ] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [ ] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [ ] I have documented any follow-up work that this PR will require, or it does not require any.
- [ ] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [ ] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b10c72990d9ad33ba1ce2f58e51ea8d9575f4516
</p>

File: src/app/Components/Bidding/Screens/ConfirmBid/index.tsx [hunk 1]
```diff
@@ -18,16 +18,14 @@ import { PriceSummary } from "app/Components/Bidding/Screens/ConfirmBid/PriceSum
...
-import { LinkText } from "app/Components/Text/LinkText"
 import { LegacyNativeModules } from "app/NativeModules/LegacyNativeModules"
 import { navigate } from "app/navigation/navigate"
 import { partnerName } from "app/Scenes/Artwork/Components/ArtworkExtraLinks/partnerName"
 import { unsafe_getFeatureFlag } from "app/store/GlobalStore"
 import NavigatorIOS from "app/utils/__legacy_do_not_use__navigator-ios-shim"
 import { Schema, screenTrack, track } from "app/utils/track"
 import { get, isEmpty } from "lodash"
-import { Box, Button, Serif, Text, Theme } from "palette"
-import { Checkbox } from "palette/elements/Checkbox"
```
Comment (pvinis) [inline src/app/Components/Bidding/Screens/ConfirmBid/index.tsx:30]: nice

File: src/app/Components/Text/Serif.tsx [hunk 2]
```diff
@@ -1,22 +0,0 @@
-import React from "react"
-import { StyleSheet, Text, TextProps } from "react-native"
-
-export default class Serif extends React.Component<TextProps, any> {
```
Comment (pvinis) [inline src/app/Components/Text/Serif.tsx:4]: 👋

**Comment by pvinis** [comment]
cleaning up 🧹 🙌

**Pull request merged**